web/src/main/kotlin/no/nav/su/se/bakover/web/routes/me/MeRoutes.kt

```diff
@@ -21,32 +20,21 @@ data class UserData(
 
 @OptIn(io.ktor.locations.KtorExperimentalLocationsAPI::class)
 internal fun Route.meRoutes() {
-    val logger = LoggerFactory.getLogger(this::class.java)
-
     get("/me") {
-        val user = call.suUserContext.user!!
         val prince = call.authentication.principal
         val roller = getGroupsFromJWT(prince).filterMap {
             Brukerrolle.fromAzureGroup(it)
         }
 
-        user.fold(
-            ifLeft = {
-                logger.info("Feil ved henting av bruker: $it")
-                call.respond(HttpStatusCode.InternalServerError, "Feil under henting av bruker")
-            },
-            ifRight = {
-                call.respond(
-                    HttpStatusCode.OK,
-                    serialize(
-                        UserData(
-                            navn = it.displayName,
-                            navIdent = it.onPremisesSamAccountName!!,
-                            roller = roller
-                        )
-                    )
+        call.respond(
```

Comment:
Hadde det gitt mening og laget en extension function så man fikk noe ala withUser {user -> ...}

Reply:
Bare for å slippe å gjøre `call.suUserContext.user`? Virker ganske unødvendig imo.